src/tree/contexts/ContextProperties.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ITreePropertyInfo } from "../settings/ITreeSettingInfo";
+
+export type ContextProperty = 'Name' | 'Description' | 'DockerEndpoint';
+
+export const contextProperties: ITreePropertyInfo<ContextProperty>[] = [
+    { property: 'Name', exampleValue: 'my-context' },
+    { property: 'Description', exampleValue: 'remote linux VM on Azure' },
+    { property: 'DockerEndpoint', exampleValue: 'my-ssh://user1@55.55.5.555' }
```

Comment:
```suggestion
    { property: 'DockerEndpoint', exampleValue: 'ssh://user1@55.55.5.555' }
```

Reply:
Not really important, but `.555` isn't allowed in an IPv4 address either, the max is 255.